### PR TITLE
run a docker stack using production vault

### PR DESCRIPTION
**Pull request by brianosburn-intel**

### CODE REVIEW

File: docs/Docker/dockerLocal_prodVault.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    volumes:
+      - private-volume:/vault/file:rw
+      - ./vault:/vault/config:rw
+    cap_add:
+      - IPC_LOCK
+    entrypoint: vault server -config=/vault/config/vault.json
+
+```
+
+2\. Create a folder named 'vault' located in scripts\docker-compose\ and add a file named vault.json with the contents below:
```
Comment (matt-primrose) [inline docs/Docker/dockerLocal_prodVault.md:25]: Looking at entrypoint above, it seems like this instruction should say "Create a folder path named '/vault/config/' and add a file named vault.json..."

Are these pointing referencing the same place?

**Review by matt-primrose — COMMENTED** [review]
Please see my comment inline.

File: docs/Docker/dockerLocal_prodVault.md (hunk 1 above)
Comment (brianosburn-intel) [reply docs/Docker/dockerLocal_prodVault.md:25]: the ./vault in the volumes section is the folder that is mounted from the host system... that maps to /vault/config in the container. Sorry about the confusion.

Comment (matt-primrose) [reply docs/Docker/dockerLocal_prodVault.md:25]: Ahh, ok.  Figured it was something like this.  I approve then 👍

**Review by matt-primrose — APPROVED** [review]

**Pull request merged**